### PR TITLE
Image installation script and notebooks

### PR DESCRIPTION
**Pull request by Farid-Karimli**
Notifications for the image install script are facilitated by [Pushover](https://pushover.net/), you need to set the PUSHOVER_API_TOKEN and PUSHOVER_USER_KEY environment variables. 

Also included the notebook where I create captions (labels) from the metadata.

### CODE REVIEW

**Comment by trgardos** [comment]
Hi @Farid-Karimli  -- should we merge this PR as is?

File: utils/image_install_parallel.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+    args = parser.parse_args()
+    country = args.country
+
+    df = pd.read_csv(GBIF_MULTIMEDIA_DATA, delimiter="\t", usecols=['gbifID', 'identifier'], on_bad_lines='skip')
+
+    if country:
+        df = df[df['countryCode'] == country]
+
+    send_notification("Image Installation", f"Starting image installation for {len(df)} images")
```
Comment (trgardos) [inline utils/image_install_parallel.py:137]: Nice idea!

**Review by trgardos — CHANGES_REQUESTED** [review]
please remove API keys

File: utils/notifications.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import requests
+import json
+import os
+
+URL = "https://api.pushover.net/1/messages.json"
+
+def send_notification(title, message):
```
Comment (trgardos) [inline utils/notifications.py:7]: Suggest that you check if API tokens are defined in the environment and only post if they are.

**Review by trgardos — CHANGES_REQUESTED** [review]

File: utils/image_install.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -l
+
+module load miniconda
+module load academic-ml/spring-2024
+
+conda activate spring-2024-pyt
+
+export PUSHOVER_API_TOKEN=a56w88zdckxutyn1kvgjae4mtn94cj
```
[the remark this answers is not in the document]
Comment (trgardos) [reply utils/image_install.sh]: Why resolved?

File: utils/notifications.py (hunk 2 above)
Comment (trgardos) [reply utils/notifications.py:8]: Why resolved?

File: utils/image_install.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (Farid-Karimli) [reply utils/image_install.sh]: My next commit handles these.

File: utils/notifications.py (hunk 2 above)
Comment (Farid-Karimli) [reply utils/notifications.py:8]: Next commit

**Review by trgardos — APPROVED** [review]

**Pull request merged**